### PR TITLE
commit 1 d

### PR DESCRIPTION
**Pull request by dulaj93**

### CODE REVIEW

File: file.2.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@ sadf
 sadfsadf
 asdf
 
-changes from dulaj
\ No newline at end of file
+new branch dulaj
```
Comment (kmchmk) [inline file.2.txt]: Use capital letters for names.

File: file.2.txt [hunk 2]
```diff
@@ -2,4 +2,4 @@ sadf
 sadfsadf
```
Comment (kmchmk) [inline file.2.txt]: Remove this line.

**Review by kmchmk — CHANGES_REQUESTED** [review]

File: file.2.txt (hunk 1 above)
Comment (dulaj93) [reply file.2.txt]: Done.

File: file.2.txt (hunk 2 above)
Comment (dulaj93) [reply file.2.txt]: Done.

**Review by dulaj93 — COMMENTED** [review]
done

**Review by kmchmk — APPROVED** [review]

**Pull request merged**